scripts/install.sh

```diff
@@ -4,6 +4,19 @@
 # FUNCTION DEFINTIONS BEGIN
 ###########################
 
+
+check_ubuntu_version() {
+    currentver=`lsb_release -r | awk '{ print $2 }'`
+    requiredver='16.04'
+    if [ "$(printf '%s\n' "$requiredver" "$currentver" | sort -V | head -n1)" = "$requiredver" ];
```

Comment:
thats a really nice logic, if its working

1. Get current version from lsb release
2. Sort required version with current version and take the head (given the sort is ascending)
3. check if the head of the sorted version is our required version

This is however only a basic test. If you are running on another debian or ubuntu distro, results might differ ;)

Have you tested it on 14.04 (fail), 16.04 (success) and 18.04 (success)?

Reply:
Not directly important to test on 16.04. I can test the function on 14.04 in virtualbox

Reply:
I have now tested it. I can verify its working, that it kicks off 14.04. But the way it does that is by running exit. I am not sure if that is user friendly. On my 14.04 it immediately exited the terminal without displaying anything to me. I removed the exit and verified that it reach the else part.